policies/submissions.md

```diff
@@ -1,49 +1,53 @@
 ---
 layout: default
-title: Submitting Your Project To CODEVID-19
+title: Submitting Your Project
 ---
 
 Read these carefully, otherwise we can't consider your team for prizing!
 
-### Deliverables
-
-1. **REQUIRED:** A link to your working app (the website URL, or a TestFlight/Google Play opt-in URL).
-2. **REQUIRED:** A link to a screenshot we can share with voters.
-3. **REQUIRED:** The [CODEVID-19 problem](rules.html#the-problems) you're addressing.
-4. **REQUIRED:** A short description and pitch for your app we can share on Twitter and social media (140 characters or less)
-5. **REQUIRED:** A long description and pitch for your app we can use to describe your app to voters and judges.
-6. **REQUIRED:** Instructions for interacting with your app (if not included in the app itself).
-7. **REQUIRED:** An email address that judges can use to follow up with the team should they have questions or concerns. This will not be shared with the public, or with voters.
-8. **REQUIRED:** A link to your project on FindCollabs
-9. **OPTIONAL:** For open-source projects, we encourage a link to the source code repository (e.g. GitHub, GitLab, Bitbucket, etc.).
-
-### Summary of the Rules
-
-**[Ensure you've read and understand the full rules!](rules.html)**
+# Rules Summary
 
 - All projects must be [registered on FindCollabs](https://findcollabs.com/hackathon/codevid-19-isp21fkqtjupchx7kjed) to be eligible
 - Team members must all be 13 years or older, and teams larger than 5 are ineligible for prizes.
 - In the spirit of hackathons, only apps that were started **on or after March 16, 00:00 UTC** will be considered for prizing.
-- IP remains your during and after the competition.
+- You and your team retain full intellectual property rights during and after the competition.
 - Teams are responsible for following the licenses associated with the libraries, source code, and assets they use
-- By submitting your project you grant the organizers, sponsors and partners the right to share the likeness of your project (screenshots, descriptions, etc.) in public. By sharing publicly we can't make any guarantees or take responsibility for how shared content is used.
+- By submitting your project, you grant the organizers, sponsors and partners the right to share information about your team and project (screenshots, descriptions, etc.) publicly. By sharing publicly, we can't make any guarantees or take responsibility for how shared content is used.
 - Organizers and judges reserve the right to disqualify entries and winners who violate the [Code of Conduct](/code-of-conduct.html) and [Rules](/policies/rules.html) outlined on this website
 
-## Weekly Prize Submission
+**Ensure that you have also read and understood the [full rules!](rules.html)**
+
+# Submission Details
+
+## Deliverables
```

Comment:
I like the new format, but let's bump this back to the top of the page. It's the first thing teams need to understand and what they'll likely want to check before submitting.

Reply:
Agreed.  I would also opt to remove the Rules Summary section altogether and only keep a reminder to read the full rules just before the Deliverables heading.  I believe it increases the content relevancy and reduces maintenance on our part.  Thoughts?

Reply:
I know, I'll remove it from here and make it the summary in the Rules page.